src/test/java/teammates/test/cases/storage/InstructorsDbTest.java

```diff
@@ -314,12 +314,20 @@ public void testUpdateInstructorByGoogleId() throws Exception {
 
         instructorToEdit.name = "New Name";
         instructorToEdit.email = "InstrDbT.new-email@email.tmt";
+        instructorToEdit.isArchived = true;
+        instructorToEdit.role = "new role";
+        instructorToEdit.isDisplayedToStudents = false;
+        instructorToEdit.displayedName = "New Displayed Name";
         instructorsDb.updateInstructorByGoogleId(instructorToEdit);
```

Comment:
An easy way to change privileges would be using the InstructorPrivileges constructor at line 104, there are a few defined settings.